RockWeb/Blocks/Crm/PersonSearch.ascx.cs

```diff
@@ -262,6 +262,10 @@ private void BindGrid()
                         }
                 }
 
+                var personIdList = people.Select( p => p.Id ).ToList();
+
+                people = personService.Queryable(true).Where( p => personIdList.Contains( p.Id ) );
```

Comment:
I actually considered this and tested it by searching for the letter "e" which returned over 14,000 records with no issues.  I've seen scenarios where Linq created a whole bunch of parameters on a query which then led to situations where it surpassed the parameter limit in SQL but this seems to work fine.